### ISSUE TITLE
Caption Annotation of LMHaze

### ISSUE THREAD

**Issue by HuaYuuu** [wangzrk/LMHaze#2]
Hi, thank you for your impressive work! I've downloaded the LMHaze dataset. However, I did not find the caption annotation corresponding to each image, when will caption annotation be released? Thank you!

**Comment by wangzrk** [comment]
I've received your email and will will reply to you as soon as possible.————————————This is an auto reply email from QQ Mail.

**Comment by wangzrk** [comment]
> Hi, thank you for your impressive work! I've downloaded the LMHaze dataset. However, I did not find the caption annotation corresponding to each image, when will caption annotation be released? Thank you!

Thank you for your interest in our work! We are currently busy with an upcoming deadline, so the caption annotations have not been finalized and uploaded yet. We plan to organize and release them shortly after the deadline. Thank you for your patience, and please stay tuned for updates on GitHub!

**Comment by HuaYuuu** [comment]
Thanks for your reply, I tried to run the testing code on LMHaze and I notice that we need all haze, GT, and logits input to get the final results. Does the logits serve as the caption annotation corresponding to each image pair? Without these data, we may not be able to run the testing code. Thank you again and I'm looking forward for your reply.

**Comment by wangzrk** [comment]
I've received your email and will will reply to you as soon as possible.————————————This is an auto reply email from QQ Mail.

**Comment by wangzrk** [comment]
> Thanks for your reply, I tried to run the testing code on LMHaze and I notice that we need all haze, GT, and logits input to get the final results. Does the logits serve as the caption annotation corresponding to each image pair? Without these data, we may not be able to run the testing code. Thank you again and I'm looking forward for your reply.

Thank you for your interest!

As stated in our paper, the logits are generated by a large multimodal model and are not related to caption annotations. For details on how the logits are generated, please refer to Section 4.2 of our paper.

We plan to release our generation code and logits data in early March, so please stay tuned!
